kbase/narrative
# DATAUP-306 Extract info tab

# Description of PR purpose/changes

The purpose of this PR was to extract the info tab, and make it useable for both the app cell widget and the bulk import cell widget. The infoTab code has been moved to `common/commonCell/tabs/infoTab.js`. The testing file has be relocated to match this destination. This PR only deals with extracting the infoTab code and ensuring both the appCell and bulkImportCell can render it. 

I left TODO items in comments to indicate when items will need to be updated when we decide on DATAUP-309.

Assumptions for this PR:
* Properly controlling tab state is NOT part of this PR
* Feeding in actual app data is NOT part of this PR


# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-306

# Testing Instructions
To test this PR complete the following steps:
- Open a narrative, navigate to the staging area, and import a file with the type `FASTQ Reads`
- A bulk cell will appear. Select the info tab on the bulk cell, you will see it populated with the fake data.


# Dev Checklist:


# Updating Version and Release Notes (if applicable)
